extension/test/e2e/credit-card-3ds-native.spec.js

```diff
@@ -13,7 +15,7 @@ const RedirectPaymentFormPage = require('./pageObjects/RedirectPaymentFormPage')
 const CreditCardNativePage = require('./pageObjects/CreditCard3dsNativePage')
 
 // Flow description: https://docs.adyen.com/checkout/3d-secure/native-3ds2/web-component
-describe('::creditCardPayment3dsNative::', () => {
+describe.skip('::creditCardPayment3dsNative::', () => {
```

Comment:
Can you give a ticket or link with your conversation with Adyen about this problem?

Reply:
it is in email. And they just reply us these days

> Hi King-Hin,
> 
> Thank you for your patience and for informing us about this new issue.
> 
> This issue most likely has to do with a systemic error that was resulting in these authentication refusals with reason "Failed to send message to DS". We had this error since yesterday afternoon but it has been resolved since. Could you please try again and let us know if the issue persists?
> 
> Looking forward to your response.
> 
> Kind regards,
> Efthymis
> Adyen Tech Support